docs/markdown/Release-notes-for-0.62.0.md

```diff
@@ -19,9 +19,9 @@ directory, that file is loaded by gdb automatically.
 
 ## Print modified environment variables with `meson devenv --dump`
 
-With `--dump` option, all envorinment variables that have been modified are
+With `--dump` option, all environment variables that have been modified are
 printed instead of starting an interactive shell. It can be used by shell
-scripts that wish to setup their environment themself.
+scripts that wish to setup their environment themselves.
```

Comment:
themself has historically been nonstandard and the generally accepted form has been themselves.

There is some shift in modern times to accept `themself` as a word, but...

Anyway, happy to drop this change if people have an opinion.